### ISSUE TITLE
IE Support

### ISSUE THREAD

**Issue by reinerBa** [DanSnow/vue-recaptcha#29]
Hi,
i would like to use your component in a website which should run in IE 10. But Promise is even not supportet in IE 11, could you write the component without "new Promise". Maybe  
`this.$nextTick(function` () {...}
is an option?

Edit: Sorry i have not seen #7 directly

**Comment by DanSnow** [comment]
It's ok.
For now, please use `es6-shim` to support.

Actually, I'm using `Promise` as `Deferred` to schedule rendering ReCAPTCHA after it load completely.
But I can implement this function by myself without `Promise`. 
It is a possible way to drop `Promise` required.

Also, there is another way and I prefer this one.
I could just assume ReCAPTCHA that have been loaded. And I can direct use it in this component.
So you must place `<script src="<ReCAPTCHA api>" defer></script>` before this component.

If you have any other suggestion, you can leave comment below.

**Comment by reinerBa** [comment]
I included es6-shim just for this task and it works :).
What do you mean with "I could just assume ReCAPTCHA that have been loaded. "? Do you suggest the script tag should not be used in head, but instead in the parent components html-template?

**Comment by DanSnow** [comment]
Since `script` without `async` attribute guarantee the order of execution.
I can just use `grecaptcha` in this component and don't need to worry about loading complete or not.
But user must place ReCAPTCHA's api before this component. 

Here is an example.
```html
<html>
  <body>
   <div id="root"></div>
    <!-- ReCAPTCHA must be placed before -->
    <script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" defer>
    </script> 
    <script src="vue.js" defer></script>
    <script src="vue-recaptcha.js defer"></script> <!-- this component -->
    <script src="main.js" defer></script>
  </body>
</html>
```

Actually, I have not try the way above. I'm not sure if it will work.
But I think I should try to drop the requirement of `Promise` and `Object.assign`.

**Comment by DanSnow** [comment]
Yay! Now HEAD doesn't require `Promise` or `Object.assign` anymore.

**Comment by DanSnow** [comment]
I have released v0.3.0, and it doesn't require `es6-shim` in IE anymore.
